<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMonitor</class>
 <widget class="QWidget" name="MainMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>992</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mahos Main Monitor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Working Dir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pathEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="pathButton">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Log</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="loglevelBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>120</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="currentText">
            <string>DEBUG</string>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <item>
            <property name="text">
             <string>NOTSET</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>DEBUG</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>INFO</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>WARNING</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ERROR</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>CRITICAL</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="clearlogButton">
           <property name="text">
            <string>Clear log</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="logEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Node State</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="stateTable"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Server Lock</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="lockTable"/>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_4">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Note</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="NotePlainTextEdit" name="noteEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1">
         <widget class="QPlainTextEdit" name="loadnoteEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="loadnoteButton">
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>←</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="commitnoteButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>→</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPlainTextEdit" name="commitnoteEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NotePlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>mahos.gui.main_monitor_widget</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pathEdit</tabstop>
  <tabstop>pathButton</tabstop>
  <tabstop>loglevelBox</tabstop>
  <tabstop>clearlogButton</tabstop>
  <tabstop>logEdit</tabstop>
  <tabstop>stateTable</tabstop>
  <tabstop>lockTable</tabstop>
  <tabstop>noteEdit</tabstop>
  <tabstop>commitnoteButton</tabstop>
  <tabstop>commitnoteEdit</tabstop>
  <tabstop>loadnoteButton</tabstop>
  <tabstop>loadnoteEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
